### PR TITLE
(refonte) Tooling revamped : Parcel2, Vue3, and new discussions + i18n

### PR DESCRIPTION
**Pull request by gmougeolle**
Pretty massive PR with a lot of changes, including breaking ones :

**Tooling**

- Now uses Parcel JS 2 with a whole new config (doc incoming !)
- Temporarily uses a "parcel-transformer-more" for `less` files, while my PR is being merged (https://github.com/parcel-bundler/parcel/pull/5686)
- Now uses Vue3 Beta build (should be out of beta "before Q4 2020" according to Vue devs so very soon®)
- Revamped the whole build process to be way simpler : Stylemark doc is automatically built along in dev & build mode
- Removed Stylelint as it was not used at all
- Bumped NodeJS version to `^14`

**JS Components**

- Rewrote existing components and plugins to use Vue3
- New discussions component with nice dark mode
- New i18n setup for translations. Keys are in french and the only language registered is `en` to prevent from having a `fr` translation file with a `key : key` syntax
- Vanilla components now use a IIFE to register even in prod mode

**a11y**

- Fixed a lot of a11y-related stuff : `aria-*` labels, input labels, buttons... Lighthouse now gives a nice score for accessibility.

**Ui-kit**

- New `.row-inline` component
- New `.badge` component
- Pagination component is now dark and pretty
- Links now have a  `cursor:pointer` and `display: inline-block` default
- Secondary buttons have a transparent background

### CODE REVIEW

File: theme/js/components/discussions/thread-reply.vue [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        subject: {
+          id: this.subjectId,
+          class: this.subjectClass,
+        },
+      };
+
+      if (this.onSubmit)
+        this.onSubmit(values)
+          .catch((err) => {
+            vm.$toasted.error("Error posting comment");
```
Comment (gmougeolle) [inline theme/js/components/discussions/thread-reply.vue:55]: TODO : Add a toast library and translate those

File: udata_gouvfr/views.py [hunk 2]
```diff
@@ -292,3 +293,8 @@ def has_apis(ctx):
 def dataset_apis(ctx):
     dataset = ctx['dataset']
     return theme.render('dataset-apis.html', apis=dataset.extras.get(APIGOUVFR_EXTRAS_KEY))
+
+# TODO : better this, redirect is not the best. How to serve it instead ?!
```
Comment (gmougeolle) [inline udata_gouvfr/views.py:297]: This is where I need some Python-fu : I want to add a `/stylemark` route that will serve the `stylemark/` directory which is located inside the theme static folder.
According to The Internet™, we should be using the `send_from_directory` function from Flask, but this require the corresponding static directory path which I do not know how to obtain.

Comment (abulte) [reply udata_gouvfr/views.py:297]: You could try to use `udata.theme.get_current_theme().static_path` but if the redirect solution works it does not look so bad.

**Review by abulte — COMMENTED** [review]
I hope everybody read your comments, I have a pleasant time every time :-)

File: udata_gouvfr/views.py (hunk 2 above)
Comment (gmougeolle) [reply udata_gouvfr/views.py:297]: I'm going to let it as is for now, it works-ish but it's not ideal : you have to go to `/_stylemark/index.html` and you get redirected to an ugly URL. We want pretty URLs !

File: theme/js/components/discussions/thread.vue [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+          :key="commentUrl(id, index)"
+        >
+          <div class="comment-meta">
+            <avatar :user="comment.posted_by"></avatar>
+            <div>
+              <strong class="author">
+                {{
+                  comment.posted_by.first_name +
+                  " " +
+                  comment.posted_by.last_name
```
[the remark this answers is not in the document]
Comment (gmougeolle) [reply theme/js/components/discussions/thread.vue]: I've factorized an `<Author />` component that can also optionally display admin badges and stuff later on

**Comment by gmougeolle** [comment]
> I hope everybody read your comments, I have a pleasant time every time :-)

Unfortunately I think it's just you and me buddy ;-)

**Pull request merged**